client/landing/jetpack-cloud/components/ignore-threat/index.tsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { Dialog } from '@automattic/components';
+import Gridicon from 'components/gridicon';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+interface Props {
+	threatId: number;
+	threatTitle: string;
+	threatDescription: string;
+	showDialog: boolean;
+	onCloseDialog: Function;
+}
+
+class IgnoreThreatDialog extends React.PureComponent< Props > {
+	render() {
+		const buttons = [
+			{ action: 'cancel', label: translate( 'Go Back' ), onClick: this.props.onCloseDialog },
+			{
+				action: 'save',
+				label: translate( 'Ignore Threat' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Ignore threat' )` *ES Score: 14*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/ignore-threat-modal)